packages/zarf-registry/chart/values.yaml

```
@@ -48,7 +48,7 @@ podDisruptionBudget:
   minAvailable: 1
 
 autoscaling:
-  enabled: true
+  enabled: false
```

```suggestion
  enabled: true
```

Would still like to keep autoscaling enabled.  I think the RWX change is a good one but autoscaling should be useable in EKS with the correct setup - see our eks distro we recently updated for nightly testing:

https://github.com/defenseunicorns/zarf/blob/main/packages/distros/eks/eks.yaml#L13

Note use of the EBS CSI driver will require `--disable-nodegroup-eviction` on teardown.

https://github.com/defenseunicorns/zarf/blob/main/packages/distros/eks/zarf.yaml#L69

---

In our tests, we weren't able to bring down nodes with the Zarf Registry Deployment due to specified Pod Disruption Budgets.

---

For now, this PR will expose a variable for custom setting the PVC accessMode